### DISCUSSION TITLE
White screen on launch

### DISCUSSION THREAD

**Discussion by kairochen** [shedaniel/RoughlyEnoughItems#1218] [Q & A] [answered]
I'm trying to install REI and I've noticed it's the only mod that is causing this issue (even with just REI, cloth, and Architectury in mods folder) where I hit launch after choosing the fabric loader and the launcher is stuck on a white screen. Does anyone have a fix to this? These are the files/versions I currently have in my mods folder:

architectury-6.3.49-fabric.jar
cloth-api-4.0.65.jar
fabric-api-0.66.0+1.19.2.jar
RoughlyEnoughItems.9.1.572.jar

[latest.log](https://github.com/shedaniel/RoughlyEnoughItems/files/10017420/latest.log)

**Comment by rozbrajaczpoziomow** [comment, marked as answer]
Install a later fabric version:
```
A potential solution has been determined:
         - Install fabric, version 0.54.0 or later.
Unmet dependency listing:
         - Mod 'Architectury' (architectury) 6.3.49 requires version 0.54.0 or later of fabric, which is missing!
 ```
(taken from latest.log)
